package.json

```
@@ -0,0 +1,46 @@
+{
+  "name": "tslint-config-seek",
+  "version": "0.0.0-development",
+  "description": "TSLint configuration used by SEEK",
+  "main": "index.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/seek-oss/tslint-config-seek.git"
+  },
+  "author": "SEEK",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/seek-oss/tslint-config-seek/issues"
+  },
+  "scripts": {
+    "commit": "git-cz",
+    "travis-deploy-once": "travis-deploy-once",
+    "semantic-release": "semantic-release"
+  },
+  "husky": {
+    "hooks": {
+      "commit-msg": "commitlint --edit --extends seek"
+    }
+  },
+  "homepage": "https://github.com/seek-oss/tslint-config-seek#readme",
+  "devDependencies": {
+    "@commitlint/cli": "^7.2.1",
+    "commitizen": "^3.0.2",
+    "commitlint-config-seek": "^1.0.0",
+    "cz-conventional-changelog": "^2.1.0",
+    "husky": "^1.1.2",
+    "semantic-release": "^15.9.17",
+    "travis-deploy-once": "^5.0.9"
+  },
+  "release": {
+    "success": false
+  },
+  "config": {
+    "commitizen": {
+      "path": "./node_modules/cz-conventional-changelog"
+    }
+  },
+  "dependencies": {
+    "tslint-config-prettier": "^1.15.0"
+  }
```

Thoughts on adding `tslint` as a peer dep?
```suggestion
  },
  "peerDependencies": {
    "tslint": ">=5"
  }
```